### ISSUE TITLE
Allow BIPs as external source.

### ISSUE THREAD

**Issue by xinbenlv** [ethereum/EIPs#6897]
### Proposed Change

In the past, many EIPs has cited BIPs, including the very [EIP-1](https://eips.ethereum.org/EIPS/eip-1), 

```
This document was derived heavily from [Bitcoin’s BIP-0001](https://github.com/bitcoin/bips) written by Amir Taaki which in turn was derived from [Python’s PEP-0001](https://peps.python.org/).
```

and you can find a full list here: 
https://www.google.com/search?q=site%3Aeips.ethereum.org+bip&rlz=1C5CHFA_enUS1050US1050&oq=site%3Aeips.ethereum.org+bip&aqs=chrome..69i57j69i58.6731j0j4&sourceid=chrome&ie=UTF-8

Most recently, on the EIP Editing Office Hour today, an EIP draft (#6808) draft intend to link to BIPs 

Since BIP is a external resources that demonstrated time of consistency (10y) and has been used by many EIP, I'd like to propose we include BIP to allowed list of external source.

FYI: author of EIP-6808 @KBTStandardAdmin

**Comment by SamWilsn** [comment]
@lightclient and I are okay with allowing BIPs as an origin.

**Comment by lightclient** [comment]
Closed with #6907
